index.html

```diff
@@ -162,26 +162,47 @@ <h2>Introduction</h2>
+			</dd>      
+      <dt><dfn>Tooltip attribute</dfn></dt>
+			<dd>
+			  <p>Any host language attribute that would result in a user agent generating a tooltip such as in response to a mouse hover in desktop user agents.</p>
+			</dd>
+    </dl>
+  </div>
+</section>
 <section id="conformance">
   <section>
     <h3>RFC-2119 Keywords</h3>
     <p>RFC-2119 keywords are formatted in uppercase and contained in a <code>strong</code> element with <code>class="rfc2119"</code>. When the keywords shown above are used, but do not share this format, they do not convey formal information in the RFC 2119 sense, and are merely explanatory, i.e., informative. As much as possible, such usages are avoided in this specification.</p>
   </section>
   <section>
     <h3>Normative and Informative Sections</h3>
-    <p>The indication whether a section is <a class="termref">normative</a> or <a class="termref">non-normative</a> (informative) applies to the entire section including sub-sections.</p>
+    <p>The indication whether a section is normative or non-normative (informative) applies to the entire section including sub-sections.</p>
     <p>Informative sections provide information useful to understanding the specification. Such sections may contain examples of recommended practice, but it is not required to follow such recommendations in order to conform to this specification.</p>
   </section>
 </section>
-<section id="mapping_additional_nd" class="normative">
+<section id="mapping_additional_nd" class="normative" data-cite="dom">
   <h2>Name and Description</h2>
-  <p>The starting point of the name and description computation is a <abbr title="Document Object Model">DOM</abbr> <a class="termref">element</a>.  The output is a flat, unstructured string that can be as simple as a single word, or a string of space-separated tokens.  Examples include <code>Save</code> and <code>Reload from disk</code>.</p>
-  <p>An important factor is the <a class="termref">element</a>'s <a class="termref">role</a>, that determines which content contributes to the name string.  Roles have a <code>nameFrom</code> <abbr title="Resource Description Framework">RDF</abbr> property, with three possible values:  </p>
+  <p>The starting point of the name and description computation is a <abbr title="Document Object Model">DOM</abbr> [=element=].  The output is a flat, unstructured string that can be as simple as a single word, or a string of space-separated tokens.  Examples include <code>Save</code> and <code>Reload from disk</code>.</p>
+  <p>An important factor is the [=element|element's=] <a class="termref">role</a>, that determines which content contributes to the name string.  Roles have a <code>nameFrom</code> <abbr title="Resource Description Framework">RDF</abbr> property, with two possible values:  </p>
```

Comment:
There really are 3 possible values...

```suggestion
  <p>An important factor is the [=element|element's=] <a class="termref">role</a>, that determines which content contributes to the name string.  Roles have a <code>nameFrom</code> <abbr title="Resource Description Framework">RDF</abbr> property, with three possible values:  </p>
```